src/auth/token-generator.ts

```diff
@@ -152,7 +152,7 @@ export class FirebaseTokenGenerator {
       );
     }
 
-    const fullDecodedToken = jwt.decode(idToken, {
+    const fullDecodedToken: any = jwt.decode(idToken, {
```

Comment:
Some of these changes are already in master. Make sure to merge with the latest master.

Reply:
Will do.